### PR TITLE
testing/font-siji: new aport

### PR DESCRIPTION
**Pull request by tanertas**
https://github.com/stark/siji/
Siji is an iconic bitmap font based on the Stlarch font with additional glyphs.

### CODE REVIEW

**Comment by tanertas** [comment]
@andypost can you take a look at this?

File: testing/font-siji/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Contributor: Taner Tas <taner76@gmail.com>
+# Maintainer: Taner Tas <taner76@gmail.com>
+pkgname=font-siji
+pkgver=20171022_git
```
Comment (prspkt) [inline testing/font-siji/APKBUILD]: please change to: pkgver=20190218_git

File: testing/font-siji/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+_pkgver="9d88311bb127b21672b2d4b43eed1ab0e494f143"
```
Comment (prspkt) [inline testing/font-siji/APKBUILD]: please set to: _pkgver="c691f200c1c66e76daa2afc9cbbd1aa39045c906"

File: testing/font-siji/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+pkgrel=0
+pkgdesc="Siji is an iconic bitmap font based on the Stlarch font with additional glyphs."
```
Comment (prspkt) [inline testing/font-siji/APKBUILD]: let's make `pkgdesc` a bit shorter `pkgdesc="Iconic bitmap font based on Stlarch with additional glyphs"`

File: testing/font-siji/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+url="https://github.com/stark/siji/"
+arch="noarch"
+license="GPL-2.0-only"
+depends="encodings font-alias mkfontscale mkfontdir fontconfig"
+#subpackages="$pkgname-dev $pkgname-doc"
```
Comment (prspkt) [inline testing/font-siji/APKBUILD]: this can be safely removed, we only install the font file

File: testing/font-siji/APKBUILD [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+package() {
+	cd "$builddir"
+	install -Dm644 pcf/siji.pcf "$pkgdir"/usr/share/fonts/misc/siji.pcf
+#	install -Dm755 view.sh "$pkgdir"/usr/share/font-siji/view.sh
+}
+
+_package() {
+	cd "$builddir"
+}
```
Comment (prspkt) [inline testing/font-siji/APKBUILD]: please remove `_package()` function

File: testing/font-siji/APKBUILD [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+options="!check" # No test suite
+source="
+	$pkgname-$pkgver.tar.gz::https://github.com/stark/siji/archive/$_pkgver.tar.gz
+	"
+builddir="$srcdir/${pkgname/font-/}-$_pkgver"
+
+package() {
+	cd "$builddir"
+	install -Dm644 pcf/siji.pcf "$pkgdir"/usr/share/fonts/misc/siji.pcf
+#	install -Dm755 view.sh "$pkgdir"/usr/share/font-siji/view.sh
```
Comment (prspkt) [inline testing/font-siji/APKBUILD]: please remove

File: testing/font-siji/APKBUILD [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+#subpackages="$pkgname-dev $pkgname-doc"
+options="!check" # No test suite
+source="
+	$pkgname-$pkgver.tar.gz::https://github.com/stark/siji/archive/$_pkgver.tar.gz
+	"
+builddir="$srcdir/${pkgname/font-/}-$_pkgver"
+
+package() {
+	cd "$builddir"
+	install -Dm644 pcf/siji.pcf "$pkgdir"/usr/share/fonts/misc/siji.pcf
```
Comment (prspkt) [inline testing/font-siji/APKBUILD]:
```suggestion
	install -Dm644 pcf/siji.pcf "$pkgdir"/usr/share/fonts/misc/siji.pcf
	
	# compress font to save space
	gzip -9 "$pkgdir"/usr/share/fonts/misc/siji.pcf
```

File: testing/font-siji/APKBUILD (hunk 5 above)
Comment (tanertas) [reply testing/font-siji/APKBUILD]: upss. so many mistakes for such small abuild.

**Comment by tanertas** [comment]
@prspkt I think this is good now.

File: testing/font-siji/APKBUILD [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+pkgname=font-siji
+pkgver=20190218_git
+_pkgver="c691f200c1c66e76daa2afc9cbbd1aa39045c906"
+pkgrel=0
+pkgdesc="Iconic bitmap font based on Stlarch with additional glyphs."
+url="https://github.com/stark/siji/"
+arch="noarch"
+license="GPL-2.0-only"
+depends="encodings font-alias fontconfig mkfontscale mkfontdir"
+makedepends="gzip"
```
Comment (prspkt) [inline testing/font-siji/APKBUILD]: @tanertas please remove. busybox provides gzip by default.

Comment (tanertas) [reply testing/font-siji/APKBUILD]: ok

**Pull request closed without merging**